### DISCUSSION TITLE
the *Event properties* section in api-event pages

### DISCUSSION THREAD

**Discussion by skyclouds2001** [mdn/mdn-community#468] [Content]
currently, many page of type `web-api-event` would includes a *Event Type* section and *Event properties* section, like https://developer.mozilla.org/en-US/docs/Web/API/Window/storage_event

however, this brings a question that since *Event properties* section is updated by hand, its content updates may be ignored or forgot when the content of the event interface updates (like https://github.com/mdn/content/pull/29760 which was found later after the first commit)

this also brings a question that if the content of the event interface updates, the content of event need also be updated too, so if the event interface is used by many pages, it means many pages need also update (like https://github.com/mdn/content/pull/30589, the update to `PointerEvent` means 14 other event pages update)

and it is confusing that only mention event properties, not mention event methods (and static properties and static methods)

see also [API event subpage template](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Page_types/API_event_subpage_template)

**Comment by teoli2003** [comment]
The reason why we added this section there is:
- users using an event needs to know what the event object properties are.

Having them on the same page helps them.

We were aware that this increase maintainance complexity, but we think that these interfaces are stable enough so that this complexity will appear very rarely.

It is a case where we put users before writers/maintainers.
